parser/float_operation.go

```diff
@@ -1,12 +1,17 @@
 package parser
 
 type FloatOperation struct {
+	BaseOperation
 	NullOperation
 }
 
 func (o *FloatOperation) get(left Operand, right Operand) (float64, float64, error) {
-	if left == nil {
-		return 0, 0, ErrEvalOperandMissing
+	if isNil(left) {
+		if o.config.NilToZeroValue {
+			left = 0
+		} else {
+			return 0, 0, ErrEvalOperandMissing
+		}
```

Comment:
Intentaria quitar los elses. Si se quiere mas declarativo la asignacion se puede pasar a una func assignZeroValue o algo asi. Como la ves?

```suggestion
		if !o.config.NilToZeroValue {
			return 0, 0, ErrEvalOperandMissing
		}
		left = 0
```

Reply:
Saque el else de todas las operations. No lo pase a una func porque se vuelve un poco mas complejo ya que hay que saber el type para poder devolverl el zero value de cada type.